src/tests/ftest/util/daos_io_conf.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2019 Intel Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+"""
+import os
+
+from apricot import TestWithServers
+from command_utils import ExecutableCommand
+from test_utils import TestPool
+
+class IoConfGen(ExecutableCommand):
+    """Base IoConf execute command class.
+
+    :avocado: recursive
+    """
+    def __init__(self, command, path="", env=None):
+        """Create a ExecutableCommand object.
+
+        Uses Avocado's utils.process module to run a command str provided.
+
+        Args:
+            command (str): string of the command to be executed.
+            path (str, optional): path to location of command binary file.
+                Defaults to "".
+        """
+        super(IoConfGen, self).__init__("", command, path)
+        self.verbose = True
+        self.env = env
+
+class IoConfTestBase(TestWithServers):
+    """Base rebuild test class.
+
+    :avocado: recursive
+    """
+
+    def __init__(self, *args, **kwargs):
+        """Initialize a IoConfTestBase object."""
+        super(IoConfTestBase, self).__init__(*args, **kwargs)
+        self.pool = TestPool(self.context, self.log)
+
+    def setUp(self):
```

Comment:
Why is this needed if there is no modification to setUp()?

Reply:
I think it's preferred to have it match with super class TestWithServers setUp(). Do you prefer to put it in side init()? or somewhere else ?